templates/resource_tracking/resource_group/resource-group-edit.html

```diff
@@ -1,137 +1,127 @@
 {% extends 'base.html' %}
-
-{% block content %}
-    <div class="content-wrapper" style="min-height: 278px;">
-        <!-- Content Header (Page header) -->
-        <div class="content-header">
-            <div class="container-fluid">
-                <div class="row mb-2">
-                    <div class="col-sm-6">
-                        {% include "generics/breadcrumbs.html" %}
-                    </div><!-- /.col -->
-                </div><!-- /.row -->
-            </div><!-- /.container-fluid -->
+{% block header_button %}
+    <a class="btn btn-default btn-sm float-right"
```

Comment:
Please switch button to class to ""btn btn-primary float-right"
And badge to "badge badge-secondary"